lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/LibLoader.java

```diff
@@ -73,22 +73,35 @@ public void loadLib() throws IOException {
     LOG.log(Level.INFO, "Done loading DLLs for Driver at time {0}." + new Date().toString());
   }
 
+  //load Org.Apache.REEF.Bridge.JavaClrBridge.dll
   private void loadMixedDLL() throws IOException {
     try {
       final String JavaClrBridgeDll = this.reefFileNames.getMixedDLLFile().getAbsolutePath();
       LOG.log(Level.INFO, "loadMixedDLL() -JavaClrBridge Dll: {0}.", JavaClrBridgeDll);
       System.load(JavaClrBridgeDll);
     } catch (final Throwable e) {
-      LOG.log(Level.WARNING, "loadMixedDLL() - Unable to load {0}, trying the JAR", this.reefFileNames.getMixedDLLFile().getAbsolutePath());
-      final String tempLoadDir = System.getProperty(USER_DIR) + this.reefFileNames.getLoadDir();
-      new File(tempLoadDir).mkdir();
-      LOG.log(Level.INFO, "loadMixedDLL() - tempLoadDir created: {0} ", tempLoadDir);
-      final String bridgeMixedDLLName = this.reefFileNames.getBridgeMixedDLLName();
-      LOG.log(Level.INFO, "loadMixedDLL() - BridgeMixedDLLName: {0}", bridgeMixedDLLName);
-      loadFromReefJar(bridgeMixedDLLName, false);
+      try {
```

Comment:
We should move these calls into separate functions. E.g. `loadBridgeDLLFromLocal()`, `loadBridgeDLLFromGlobal()` and `loadBridgeDLLFromJAR()`. That way, the control flow in this method can be cleaned up.